src/aliceVision/feature/imageStats.hpp

```diff
@@ -0,0 +1,89 @@
+// This file is part of the AliceVision project.
+// Copyright (c) 2016 AliceVision contributors.
+// Copyright (c) 2012 openMVG contributors.
+// This Source Code Form is subject to the terms of the Mozilla Public License,
+// v. 2.0. If a copy of the MPL was not distributed with this file,
+// You can obtain one at https://mozilla.org/MPL/2.0/.
+
+#pragma once
+
+#include <aliceVision/image/all.hpp>
+
+
+namespace aliceVision {
+namespace feature {
+
+/**
+ * @brief Compute Contrast Factor
+ * @param[in] image Input image for the given octave
+ * @param[in] percentile
+ * @return contrastFactor
+ */
+inline float computeAutomaticContrastFactor(const image::Image<float>& image, const float percentile)
```

Comment:
`alicevision_feature` is not a header only lib anymore, we can split the code, even if it is just one function